docs/cloud/satellites/gha-runners.md

```diff
@@ -1,66 +1,48 @@
-
-#### Expiration time
-Important: The token is used during the configuration process and _each time_ a job event is received -- be sure to set the expiration time accordingly.
+Satellite-based GitHub Actions runners can be enabled for a particular repository or for all repositories of a GitHub organization at once.
 
-#### Permissions
-##### Organization-wide integration
-For organization-wide integrations, the user must be an admin of the organization and the token must have the following scopes/permissions:
+The integration process requires you to provide us with a GitHub token, so we can:
+- register a webhook in your GitHub repository/organization to receive events associated to GitHub Actions jobs
+- create GitHub self-hosted runners on demand, to process your repository/organization jobs
 
-Personal access tokens (classic) scopes:
-- `admin:org_hook`
-- `admin:org`
+Follow the next steps to create such integrations:
 
-Fine grained token permissions: 
-- `organization_hooks:write`
-- `organization_self_hosted_runners:write`
+### 1. Request early access
+This feature is in closed-beta at the moment. You can request early access through support@earthly.dev
 
-##### Repository integration
-For repository integrations, the user must be an admin of the repository and the token must have the following scopes/permissions:
+### 2. Create a GitHub token
+Both GitHub classic and fine-grained tokens are supported, but depending on the type of installation (organization-wide or for a single repository), the provided token requires different scopes. 
 
-Personal access tokens (classic) scopes:
-- `admin:repo_hook`
-- `repo`
+| Integration type  | User type          | Classic token scopes          | Fine-grained token permissions                                       | 
+|-------------------|--------------------|-------------------------------|----------------------------------------------------------------------|
+| Organization-wide | Organization admin | `admin:org_hook`, `admin:org` | `organization_hooks:write`, `organization_self_hosted_runners:write` |
+| Single repository | Repository admin   | `admin:repo_hook`, `repo`     | `repository_hooks:write`, `administration:write`                     |
 
-Fine grained token permissions:
-- `repository_hooks:write`
-- `administration:write`
+{% hint style='info' %}
+Follow the [official docs](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens) for detailed information on how to create a GitHub token, and make sure to set an expiration long enough, since the integration won't work after the token expires. 
+{% endhint %}
 
-### CLI
+### 3. Register via CLI
+Create the integration using the `earthly github add` CLI command, passing the token created in the previous step. 
 
 `$ earthly github add --help`
```

Comment:
This should be the command the user actually runs. I would avoid just pasting the help text, and instead rewriting the help text here to explain it better. For example:

This command sets the configuration to create a new GitHub-Earthly integration, to trigger satellite builds from GitHub Actions (GitHub Actions).

```
earthly github add --org <earthly-org> --gh-org <github-org> --gh-repo <github-repo> --gh-token <token>
```

Where
* `earthly-org` is ...
* `github-org` is ...


(If there are multiple flavors of this command, you could just have a few code blocks like this to explain each flavor separately - each flavor would have different flags)